com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.compute

```diff
@@ -98,6 +99,20 @@ groupshared int gs_localLightList[GROUP_SIZE_1D*GROUP_SIZE_1D][MAX_SUPPORTED_LIG
 
 #endif
 
+real3 EvaluateVolumetricAmbientProbe(real3 normalWS)
```

Comment:
it miss a comment to say that _VolumetricAmbientProbeBuffer include the cornel shank case. Just so people don't ask themselve why there is a differnt funciton EvaluateAmbientProbe